easybuild/easyconfigs/m/Mothur/Mothur-1.48.2_fix-hardcoding.patch

```diff
@@ -0,0 +1,118 @@
+diff -ruN mothur-1.48.2.orig/Makefile /tmp/eb/Mothur/1.48.2/foss-2023b/mothur-1.48.2/Makefile
+--- mothur-1.48.2.orig/Makefile	2024-11-04 20:53:24.000000000 +0100
++++ /tmp/eb/Mothur/1.48.2/foss-2023b/mothur-1.48.2/Makefile	2024-11-15 10:25:46.139186951 +0100
+@@ -20,7 +20,7 @@
+ # GSL_INCLUDE_DIR - location of GSL include files
+ # MOTHUR_FILES - The MOTHUR_FILES parameter is optional, but allows you to set a default location for mothur to look for input files it can't find. This is often used for reference files you want to store in one location separate from your data.
+ 
+-INSTALL_DIR ?= "\"Enter_your_mothur_install_path_here\""
++INSTALL_DIR ?= "%(installdir)s"
```

Comment:
Hm, is this tested to work? I don't see how it could, the %(installdir)s templates are not applied to patches.

Looking at the custom easyblock that this thing unfortunately needs to use, it's just manually making the installation via copying files, so I this probably doesn't even have any impact.